xbmc/pvr/epg/EpgContainer.cpp

```diff
@@ -760,7 +766,7 @@ bool CPVREpgContainer::UpdateEPG(bool bOnlyPending /* = false */)
     progressHandler->DestroyProgress();
 
   for (const auto& epg : invalidTables)
-    DeleteEpg(epg);
+    DeleteEpg(epg, true);
```

Comment:
@phunkyfish  do we have a chance to get rid of this (last) sync delete?

Reply:
Sorry, yes, that's an obvious change. I didn't see it at all!

Reply:
Ok, latest push should add that remaining case.